core/templates/css/oppia.css

```diff
@@ -884,6 +884,13 @@ textarea {
   box-shadow: 0 6px 12px rgba(0, 0, 0, .175);
 }
 
+@media (max-width: 740px) {
+  .dropdown-menu {
```

Comment:
This will affect all dropdown menus on the Oppia site. Is that expected?

Reply:
@ankita240796 I have tested it thoroughly .

Reply:
Hmm, I am still not sure. Can you share a doc with all the testing that you performed?

@seanlip any suggestions here?

Reply:
As @ankita240796 said, please look for all instances of this class in the codebase and provide a doc with screenshots of all these cases. We need to be very careful when modifying CSS rules that apply globally.

On the other hand, if you want your CSS rules to affect a specific element, it is important to scope them down to that particular element. (Please note this for both this and future PRs.)

Thanks!